graphrag/query/api.py

```diff
@@ -252,6 +262,11 @@ async def local_search_streaming(
     vector_store_type = vector_store_args.get("type", VectorStoreType.LanceDB)
 
     _entities = read_indexer_entities(nodes, entities, community_level)
+
+    base_dir = Path(cast(str, root_dir)) / Path(config.storage.base_dir)
+    resolved_base_dir = resolve_timestamp_path(base_dir)
+    lancedb_dir = os.path.join(resolved_base_dir, "lancedb")  # noqa: PTH118
```

Comment:
since  resolved_base_dir is already a Path, you can just do resolved_base_dir / "lancedb"
The / operator behaves specially when working over paths and strings

Reply:
Also same with other concats. It is not that you're doing it wrong, but you can just leverage how / behaves when one of the variables is already a Path object